lib/Command/FixEncryptedVersion.php

```diff
@@ -63,6 +63,12 @@ protected function configure() {
 				'p',
 				InputArgument::OPTIONAL,
 				'Limit files to fix with path, e.g., --path="/Music/Artist". If path indicates a directory, all the files inside directory will be fixed.'
+			)->addOption(
+				'increment-range',
+				'i',
+				InputArgument::OPTIONAL,
+				'Try versions within a given upper bound',
+				5
```

Comment:
Q1: the option says: `increment-range` and the text tells `upper bound` this is confusing.
Q2: on line 71, there is the number 5 as part of the option, where there are none of these at other options: Is this a default value (and if yes), what does it mean?

Reply:
Yes this is a default. The command will try all versions from (-5 to +5)

Reply:
You can optionally specify the value via `-i or --increment-range` option. If nothing is specified the default 5 is used which was hardcoded before.

Here is an explanation: https://github.com/owncloud/docs/issues/3594#issuecomment-850232769

Reply:
You can optionally specify the value via `-i or --increment-range` option. If nothing is specified the default 5 is used which was hardcoded before.

Here is an explanation: https://github.com/owncloud/docs/issues/3594#issuecomment-850232769

Reply:
Not sure how to elegantly describe this value. Also edited my comment here: https://github.com/owncloud/encryption/pull/274#discussion_r641464199
```
~/c/core:λ ./occ encryption:fix-encrypted-version --help
Description:
  Fix the encrypted version if the encrypted file(s) are not downloadable.

Usage:
  encryption:fix-encrypted-version [options] [--] <user>

Arguments:
  user                                   The id of the user whose files need fixing

Options:
  -p, --path=PATH                        Limit files to fix with path, e.g., --path="/Music/Artist". If path indicates a directory, all the files inside directory will be fixed.
  -i, --increment-range=INCREMENT-RANGE  Try versions within a given upper bound [default: "5"]
  -h, --help                             Display this help message
  -q, --quiet                            Do not output any message
  -V, --version                          Display this application version
      --ansi                             Force ANSI output
      --no-ansi                          Disable ANSI output
  -n, --no-interaction                   Do not ask any interactive question
      --no-warnings                      Skip global warnings, show command output only
  -v|vv|vvv, --verbose                   Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
```

Reply:
What about:
'Find the correct version of the file to verify the signature. Searches in increments from -n to +n.'